### PR TITLE
ign -> gz Shared Lib Migration : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/744

### CODE REVIEW

**Review by chapulina — COMMENTED** [review]
Just a minor comment, LGTM otherwise

File: Migration.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ but with improved human-readability..
 - Header files under `ignition/...` are deprecated and will be removed in future versions.
   Use `gz/...` instead.
 
+* The shared libraries have `gz` where there used to be `ignition`, and also have their `lib` and `.so` prefixes and suffixes removed.
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply Migration.md]: Fixed!

[the remark this answers is not in the document]
Comment (chapulina) [reply Migration.md]: I still see the entry though

[the remark this answers is not in the document]
Comment (methylDragon) [reply Migration.md]: Hmm, how about now?

**Review by chapulina — APPROVED** [review]

**Pull request merged**